backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/Context.kt

```diff
@@ -429,3 +429,12 @@ private fun MemberScope.getContributedClassifier(name: String) =
 
 private fun MemberScope.getContributedFunctions(name: String) =
         this.getContributedFunctions(Name.identifier(name), NoLookupLocation.FROM_BUILTINS)
+
+internal class LogBuilder(val context: Context) {
+    operator fun String.unaryPlus() = context.log { this }
+}
+
+internal fun Context.buildLog(messageBuilder: LogBuilder.() -> Unit) {
```

Comment:
`logMultiple`, `logBatch`?

Reply:
+1, `buildX` is used for functions that produce value of type X.